### ISSUE TITLE
Charts did not respect View's padding system

### ISSUE THREAD

**Issue by chenFlyingkite** [PhilJay/MPAndroidChart#3329]
I take the example usage of com.github.mikephil.charting.charts.LineChart

And I follow the Getting Started to add my data
https://github.com/PhilJay/MPAndroidChart/wiki/Getting-Started

And then I just want to set the view's padding inside.
But after I run in Android devices, it did not respect the paddings and still draw with whole width/height

As in source code of Chart
com.github.mikephil.charting.charts.Chart
Its onDraw() did not takes the padding left/top/right/bottom into consideration.
(Actually the whole file of Chart did not use it)

**Comment by HarryMMR** [comment]
Yes. It must be a bug. I face the same issue too. Any solution for this?
